### DISCUSSION TITLE
[@types/umzug] depends on deprecated types package

### DISCUSSION THREAD

**Discussion by wSedlacek** [DefinitelyTyped/DefinitelyTyped#57134] [Issues with a @types package]
I am not sure if this is normal, but I am attempting to clean out my deprecated packages and I noticed that `@types/umzug` is still depending on `@types/mongodb` despite it being deprecated. Is this expected?

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/2026ab9fbd33457124a0a6a55d5c1d1e3f696764/types/umzug/package.json#L4

```
npm WARN deprecated @types/mongodb@4.0.7: mongodb provides its own types. @types/mongodb is no longer needed.
```

```
└─┬ @mikro-orm/cli@4.5.9
  └─┬ @mikro-orm/migrations@4.5.9
    └─┬ @types/umzug@2.3.0
      └── @types/mongodb@4.0.7
```

[a comment by an automated account is not shown]

**Comment by emmanuelgautier** [comment]
You are right. The newest package provides its own types. The following PR has been created according to this new version: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/57136

Thanks for your report!
